internal/overlord/servstate/handlers.go

```diff
@@ -654,6 +659,7 @@ func (s *serviceData) terminateTimeElapsed() error {
 		if err != nil {
 			logger.Noticef("Cannot send SIGKILL to process: %v", err)
 		}
+
 		s.transitionRestarting(stateKilling, s.restarting)
 		time.AfterFunc(failWait-killWait, func() { logError(s.killTimeElapsed()) })
```

Comment:
We'll need to also update the other uses of `killWait` (there's one here and also in `checkFailed`). Perhaps we can have a `s.gracePeriod()` method that gets the plan grace-period or default if not set, to keep that logic in one place?

We also need to change how the `failWait` constant works. It should probably be an additional duration (5 seconds) after the grace period, rather than being the total time (you might have to update the tests). Maybe call it something meaningful for the new usage, like `timeBeforeKill`?

Reply:
I have called it `killWaitDuration` as that is what it's doing. Changed to 5 seconds as that is the amount of time afforded previously and it's been working for Pebble thus far.